# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1320462542 25200
#      Fri Nov 04 20:09:02 2011 -0700
# Node ID 822f80b908660755b800fc6023b7ef4195a17c1c
# Parent  847812137666c0ac1a7553004e62f94f82752043
unimplemented.m: Remove functions which have been implemented.

* unimplemented.m: Remove functions which have been implemented.

diff --git a/scripts/help/unimplemented.m b/scripts/help/unimplemented.m
--- a/scripts/help/unimplemented.m
+++ b/scripts/help/unimplemented.m
@@ -76,17 +76,16 @@ endfunction
 
 function list = missing_functions ()
   persistent list = {
   "DelaunayTri",
   "MException",
   "RandStream",
   "TriRep",
   "TriScatteredInterp",
-  "addpref",
   "align",
   "alim",
   "alpha",
   "alphamap",
   "annotation",
   "audiodevinfo",
   "audioplayer",
   "audiorecorder",
@@ -95,17 +94,16 @@ function list = missing_functions ()
   "auwrite",
   "avifile",
   "aviinfo",
   "aviread",
   "bar3",
   "bar3h",
   "bench",
   "betaincinv",
-  "bicg",
   "bicgstabl",
   "brush",
   "builddocsearchdb",
   "bvp4c",
   "bvp5c",
   "bvpget",
   "bvpinit",
   "bvpset",
@@ -188,17 +186,16 @@ function list = missing_functions ()
   "frame2im",
   "freqspace",
   "funm",
   "gallery",
   "gammaincinv",
   "gco",
   "getframe",
   "getpixelposition",
-  "getpref",
   "gmres",
   "grabcode",
   "graymon",
   "gsvd",
   "guidata",
   "guide",
   "guihandles",
   "handle",
@@ -233,17 +230,16 @@ function list = missing_functions ()
   "inspect",
   "instrfind",
   "instrfindall",
   "interpstreamspeed",
   "iscom",
   "isinterface",
   "isjava",
   "isocaps",
-  "ispref",
   "isstudent",
   "javaArray",
   "javaMethod",
   "javaMethodEDT",
   "javaObject",
   "javaObjectEDT",
   "javaaddpath",
   "javachk",
@@ -334,25 +330,23 @@ function list = missing_functions ()
   "quad2d",
   "questdlg",
   "rbbox",
   "recycle",
   "reducepatch",
   "reducevolume",
   "resample",
   "rgbplot",
-  "rmpref",
   "root",
   "rotate",
   "rotate3d",
   "selectmoveresize",
   "sendmail",
   "serial",
   "setpixelposition",
-  "setpref",
   "showplottool",
   "shrinkfaces",
   "smooth3",
   "snapnow",
   "sound",
   "soundsc",
   "ss2tf",
   "stream2",
@@ -397,24 +391,22 @@ function list = missing_functions ()
   "uitable",
   "uitoggletool",
   "uitoolbar",
   "uiwait",
   "undocheckout",
   "unicode2native",
   "unloadlibrary",
   "unmesh",
-  "usejava",
   "userpath",
   "validateattributes",
   "verLessThan",
   "viewmtx",
   "visdiff",
   "volumebounds",
-  "waitbar",
   "waitfor",
   "warndlg",
   "waterfall",
   "wavfinfo",
   "wavplay",
   "wavrecord",
   "web",
   "whatsnew",
